package/src/index.ts

```diff
@@ -278,6 +298,36 @@ export class CandleClient {
     }
   }
 
+  private toInternalTradeAsset(asset: TradeAsset): InternalTradeAsset {
+    switch (asset.assetKind) {
+      case "fiat":
+        return { fiatAsset: asset };
+      case "stock":
+      case "crypto":
+        return {
+          marketTradeAsset: {
+            ...asset,
+            assetKind: asset.assetKind as "stock" | "crypto",
```

Comment:
#req don't use `as` in TS

Reply:
Added fixme, but already doing this in a few spots.